### PR TITLE
SSS work / experiments.

### PR DESCRIPTION
**Pull request by est77**
- Added a BSSRDF sample class.
- Added an empty directional dipole BSSRDF model.
- Added some SSS texture mapping code.
- Added BSSRDFs to python.

### CODE REVIEW

File: src/appleseed/CMakeLists.txt [hunk 1]
```diff
@@ -1284,7 +1284,10 @@ set (renderer_modeling_bssrdf_sources
     renderer/modeling/bssrdf/bssrdf.h
     renderer/modeling/bssrdf/bssrdffactoryregistrar.cpp
     renderer/modeling/bssrdf/bssrdffactoryregistrar.h
+    renderer/modeling/bssrdf/bssrdfsample.h
     renderer/modeling/bssrdf/bssrdftraits.h
+    renderer/modeling/bssrdf/dirpolebssrdf.cpp
```
Comment (dictoon) [inline src/appleseed/CMakeLists.txt]: Woops, "dirpole" -> "dipole"

File: src/appleseed/renderer/modeling/bssrdf/bssrdf.cpp [hunk 2]
```diff
@@ -35,9 +35,91 @@
 
 using namespace foundation;
 
+/*
+
+Quick ref:
+----------
+
+    sigma_a     absorption coeff.;
```
Comment (dictoon) [inline src/appleseed/renderer/modeling/bssrdf/bssrdf.cpp]: Nice, but remove the semicolons :)

File: src/appleseed/CMakeLists.txt (hunk 1 above)
Comment (est77) [inline src/appleseed/CMakeLists.txt]: it's directional dipole -> dirpole

Comment (dictoon) [inline src/appleseed/CMakeLists.txt]: Oh ok... In that case I'd strongly prefer to spell it out completely: directionaldipolebssrdf.h... Not *that* long, and so much clearer :)

File: src/appleseed/renderer/modeling/bssrdf/bssrdfsample.h [hunk 3]
```diff
@@ -0,0 +1,95 @@
+
+//
+// This source file is part of appleseed.
+// Visit http://appleseedhq.net/ for additional information and resources.
+//
+// This software is released under the MIT license.
+//
+// Copyright (c) 2014-2015 Esteban Tovagliari, The appleseedhq Organization
```
Comment (dictoon) [inline src/appleseed/renderer/modeling/bssrdf/bssrdfsample.h]: Just 2015, probably.

File: src/appleseed/renderer/modeling/bssrdf/dirpolebssrdf.h [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+
+#ifndef APPLESEED_RENDERER_MODELING_BSSRDF_DIRPOLE_BSSRDF_H
```
Comment (dictoon) [inline src/appleseed/renderer/modeling/bssrdf/dirpolebssrdf.h]: No underscore between `DIRPOLE` and `BSSRDF`.

**Pull request merged**